src/.vuepress/config/createNavbar.ts

```diff
@@ -1,4 +1,4 @@
-import {MyPaNavLink, MyPaNavbarConfigArray} from '@mptheme/config.types';
+import {type MyPaNavLink, type MyPaNavbarConfigArray} from '@mptheme/config.types';
```

Comment:
This change should not be in this pr? Maybe something that is automatically changed but I would still remove it, if possible.